src/unity/toolkits/drawing_recognition/data_preparation.cpp

```diff
@@ -0,0 +1,262 @@
+/* Copyright © 2017 Apple Inc. All rights reserved.
```

Comment:
Please leave comments/feedback for the `src/unity/toolkits/drawing_recognition` code path in #1449 since that is more up to date.